configuration/debian/tedge/postrm

```diff
@@ -1,8 +1,69 @@
 #!/bin/sh
 set -e
 
-### Remove configs
+remove_user_tedge() {
+    if getent passwd tedge > /dev/null; then
+        deluser --quiet --system tedge
+    fi
+}
 
-if [ "$1" = "purge" ]; then
-    rm -rf /etc/tedge
-fi
+remove_tedge_users_group() {
+    if getent group tedge-users > /dev/null; then
+        groupdel tedge-users
+    fi
+}
+
+purge_configs() {
+    if [ -d "/etc/tedge" ]; then
+        rm -rf /etc/tedge
+    fi
+}
+
+purge_sudoers_file() {
+    if [ -f "/etc/sudoers.d/tedge-users" ]; then
+        rm /etc/sudoers.d/tedge-users
+    fi
+
+    if [ -f "/etc/sudoers.d/010_pi-nopasswd" ]; then
+        rm /etc/sudoers.d/tedge-users-nopasswd
+    fi
+}
+
+remove_mosquitto_edit() {
+    if [ -f "/etc/mosquitto/mosquitto.conf" ]; then
+        sed -i '/include_dir \/etc\/tedge\/mosquitto-conf/d' /etc/mosquitto/mosquitto.conf
+    fi
+}
+
+purge_var_log() {
+    if [ -d "/var/log/tedge" ]; then
+        rm -rf /var/log/tedge
+    fi
+}
+
+case "$1" in
+    purge)
+        remove_user_tedge
+        remove_tedge_users_group
+        remove_mosquitto_edit
+        purge_sudoers_file
```

Comment:
We also need to remove the sudoers file in the `remove` case.

Otherwise a user created with the `tedge` name will gain sudo power for free.